awx/ui/test/e2e/objects/applications.js

```diff
@@ -56,8 +56,9 @@ module.exports = {
             if (application.redirectUris) {
                 this.section.add.setValue('@redirectUris', application.redirectUris);
             }
+            this.section.add.click('@save'); // flake avoidance. triple click ensures it works.
```

Comment:
I wish we had waitForBeingClickable or something like that

Reply:
@elyezer this is a feature that's better handled in the newest version of nightwatch.